dust_extinction/averages.py

```diff
@@ -121,6 +122,108 @@ def evaluate(self, in_x):
         return f(x)
 
 
+class B92_MWAvg(BaseExtModel):
+    r"""
+    Bastiaansen (1992) Optical Extinction Curve
+
+    Parameters
+    ----------
+    None
+
+    Raises
+    ------
+    None
+
+    Notes
+    -----
+    From Bastiaansen (1992, A&AS, 93, 449-462)
+
+    Example showing the average curve
+
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        import matplotlib.pyplot as plt
+        import astropy.units as u
+
+        from dust_extinction.averages import B92_MWAvg
+
+        fig, ax = plt.subplots()
+
+        # define the extinction model
+        ext_model = B92_MWAvg()
+
+        # generate the curves and plot them
+        x = np.arange(1.0/ext_model.x_range[1], 1.0/ext_model.x_range[0], 0.1) * u.micron
+
+        ax.plot(x,ext_model(x),label='B1992')
+        ax.plot(1.0/ext_model.obsdata_x, ext_model.obsdata_axav, 'ko',
+                label='obsdata')
+
+        ax.set_xlabel(r'$\lambda$ [$\mu m$]')
+        ax.set_ylabel(r'$A(x)/A(V)$')
+
+        ax.legend(loc='best')
+        plt.show()
+    """
+
+    x_range = [1.0 / 0.7873, 1.0 / 0.3402]
+
+    Rv = 3.1  # assumed!
+
+    # fmt: off
+    obsdata_x = 1.0/np.array(
```

Comment:
You need to put a space before and after the `/`.  That should fix the formatting error.

Reply:
Not picked up by `black` as the autoformatting is turned off to avoid having it make a separate line for *each* number.